cypress/integration/brewery/CreateScenario-tests.spec.js

```diff
@@ -123,9 +123,14 @@ describe('Create scenario', () => {
       }
     );
 
+    // Check parameters accordion state after creating scenario
+    ScenarioParameters.getParametersTabs().should('be.visible');
+    ScenarioManager.switchToScenarioManager();
+    Scenarios.switchToScenarioView();
+    ScenarioParameters.getParametersTabs().should('be.visible');
+
     // Edit master paramameters values
     ScenarioParameters.edit();
-    ScenarioParameters.getParametersTabs().should('be.visible');
```

Comment:
We should probably not remove this check from the parameters edition test

Reply:
We could... but the same check has been done some lines above.